content/courses/onchain-development/intro-to-onchain.md

```diff
@@ -1,44 +1,57 @@
 ---
-title: Intro to onchain development
+title: Intro to Solana Onchain Development
 objectives:
   - Understand how Solana onchain programs work
-  - Describe the structure and operation of onchain programs
+  - Know about the structure and operation of Solana programs
   - Build a basic program
 description:
-  "How onchain programs (often called 'smart contracts') work on Solana."
+  "Discover how onchain programs ( often called 'smart contracts') work on
+  Solana and learn to build your own."
 ---
 
 ## Summary
 
-- **Onchain programs** are programs that run on Solana.
-- Programs have one or more **instruction handlers**. Instruction handlers are
-  functions that process the **instructions** in Solana transactions.
-- Instruction handlers write their data to Solana **accounts** and then either
-  succeed or fail.
-- Solana programs are most commonly written in **Rust**, typically with the
-  **Anchor** framework.
-- Anchor programs have IDLs, which are small JSON files describing programs.
-  IDLs can be used to automatically create JS/TS libraries for a program.
+- **Onchain programs** run directly on Solana, just like programs on your
```

Comment:
```suggestion
- **Onchain programs** (sometimes called 'smart contracts') run directly on Solana, just like programs on your
```

Reply:
We recently started to use 'smart contract' just for SEO and recognizability purposes.